mps_youtube/players/mplayer.py

```diff
@@ -0,0 +1,230 @@
+import os
+import sys
+import tempfile
+import subprocess
+import json
+import re
+import socket
+import time
+
+from .. import g, screen, c, paths, config, util
+
+from ..player import Player
+
+mswin = os.name == "nt"
+not_utf8_environment = mswin or "UTF-8" not in sys.stdout.encoding
+
+
+class Mplayer(Player):
+    def _generate_real_playerargs(self, song, override, stream, isvideo, softrepeat):
+        """ Generate args for player command.
+
+        Return args.
+
+        """
+
+        if "uiressl=yes" in stream['url'] and "mplayer" in config.PLAYER.get:
+            ver = g.mplayer_version
+            # Mplayer too old to support https
+            if not (ver > (1,1) if isinstance(ver, tuple) else ver >= 37294):
+                raise IOError("%s : Sorry mplayer doesn't support this stream. "
+                              "Use mpv or update mplayer to a newer version" % song.title)
+
+        args = config.PLAYERARGS.get.strip().split()
+
+        known_player = util.is_known_player(config.PLAYER.get)
+        if known_player:
+            pd = g.playerargs_defaults[known_player]
+            args.extend((pd["title"], song.title))
```

Comment:
As in #752, not encapsulating the title string in quotes seems to break playback on some systems. Maybe we could fix this at the same time by making sure all strings passed as arguments have quotation marks around them?

Reply:
Done!